core/src/main/java/org/elasticsearch/tribe/TribeService.java

```diff
@@ -154,6 +156,12 @@ public static Settings processSettings(Settings settings) {
     public static final Set<String> TRIBE_SETTING_KEYS = Sets.newHashSet(TRIBE_NAME_SETTING.getKey(), ON_CONFLICT_SETTING.getKey(),
             BLOCKS_METADATA_INDICES_SETTING.getKey(), BLOCKS_METADATA_SETTING.getKey(), BLOCKS_READ_INDICES_SETTING.getKey(), BLOCKS_WRITE_INDICES_SETTING.getKey(), BLOCKS_WRITE_SETTING.getKey());
 
+    // these settings should be passed through to each tribe client, if they are not set explicitly
+    private static final List<String> PASS_THROUGH_SETTINGS = Arrays.asList(
+        NetworkService.GLOBAL_NETWORK_HOST_SETTING.getKey(),
+        NetworkService.GLOBAL_NETWORK_BINDHOST_SETTING.getKey(),
+        NetworkService.GLOBAL_NETWORK_PUBLISHHOST_SETTING.getKey()
```

Comment:
I have no idea what else we may need also, it is not the first time we forget about some needed settings... looking back I am not sure that the approach of selecting settings is correct, judging from how many problems it caused to users. It may be cleaner but the truth is that it is easier to define what settings don't make sense in a tribe client node and exclude them rather than having to include all of the settings that people may need passed through. My 2 cents.